### DISCUSSION TITLE
ant design sidebar layout left to right shifting

### DISCUSSION THREAD

**Discussion by opuopu** [ant-design/ant-design#49328] [General]
how can i shift ant design sidebar layout left to right when rtl is true.
![image](https://github.com/ant-design/ant-design/assets/43672649/0b6c025f-0822-44c7-b3b2-be6bd169a22b)

this is my layout code 
![image](https://github.com/ant-design/ant-design/assets/43672649/47d7cb44-e137-40bd-bf0f-fdb729aaf837)

**Comment by AlEkramHossainAbir** [comment]
Use <Sidebar /> Sidebar component after its sibling <Layout /> Layout component
